libs/mark_predictions_on_image.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pandas as pd
+from draw_utils import place_markers_on_images
+from utils import list_dirs
+
+test_data_dir = os.path.dirname(os.path.realpath(__file__)) + "/../test"
+ml_dir = "{}/tmp/pipeline2/machine_learning".format(test_data_dir)
+
+
+def get_ml_predictions_dirs(directory):
+    ml_dir_sets = list_dirs(directory)
+    return ["{}/{}".format(ml_dir, x) for x in ml_dir_sets]
+
+
+def prepare_data_from_predictions_csv(csv_dir_path):
+    df = pd.read_csv("{}/models_predictions.csv".format(csv_dir_path))
+    df = df[~df['dataset'].str.startswith('overall')]  # drop rows which starts with overall...
+    unique_files = df['landmark_image'].unique()  # find unique image filenames
+    data_to_draw = []
+
+    for u_file in unique_files:  # prepare data for drawing function
+        u_file_df = df.loc[df["landmark_image"] == u_file]
+        file_path = "{}/{}".format(test_data_dir, u_file)
```

Comment:
why is test_data_dir required here? csv path should already contain all the path we need?